controller_manager:
  ros__parameters:
    update_rate : 100
    arm_controller:
      type: forward_command_controller/ForwardCommandController


    # gripper_controller:
    #   type: joint_trajectory_controller/JointTrajectoryController
      #type: forward_command_controller/ForwardCommandController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster 


arm_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5

    interface_name: position
  

    open_loop_control: true
    allow_integration_in_goal_trajectories: true


# gripper_controller:
#   ros__parameters:
#     joints:
#       - joint_R_gear     
#     command_interfaces:
#       - position
#     state_interfaces:
#       - position
#       - velocity

    


